WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalMigrationContentResolver.kt

```diff
@@ -30,12 +30,12 @@ private fun ContentResolver.query(
 
 
 class LocalMigrationContentResolver @Inject constructor(
-    private val contextProvider: ContextProvider,
-    private val wordPressPublicData: WordPressPublicData,
-    private val queryResult: QueryResult,
+    val contextProvider: ContextProvider,
```

Comment:
I just wanted to confirm if we really need to remove `private` modifier from these parameters.

Reply:
Sadly, due to the way these inline calls are chained, it seems we need to increase the visibility. I'd be happy if there was a way to keep them private though, if you have a suggestion :thinking:

Reply:
So, it turns out there is a way :smile: We can use an annotation to make these less visible in Kotlin (though the names are still visible in the JVM). This is an improvement though :+1: I've addressed this in this commit: https://github.com/wordpress-mobile/WordPress-Android/pull/17399/commits/4db652422bcb5847cc02deb10ebc21dcc5bad907